openmoc/openmoc.i

```diff
@@ -53,6 +53,10 @@
 %warnfilter(506) log_printf(logLevel level, const char *format, ...);
 %warnfilter(511) swig::SwigPyIterator;
 %warnfilter(511) Cell::setFill;
+%ignore setFSRsToMaterialIDs(std::vector<int> FSRs_to_material_IDs);
```

Comment:
Can you update lines 56-58 to reflect the recent changes in PR #144? I'm not sure why those are not showing up as a merge conflict in this PR.